mlflow/store/_unity_catalog/registry/rest_store.py

```diff
@@ -249,6 +254,39 @@ def delete_registered_model_tag(self, name, key):
         _raise_unsupported_method(method="delete_registered_model_tag")
 
     # CRUD API for ModelVersion objects
+    def _finalize_model_version(self, name, version):
+        """
+        Finalize a UC model version after its files have been written to managed storage,
+        updating its status from PENDING_REGISTRATION to READY
+        :param name: Registered model name
+        :param version: Model version number
+        :return Protobuf ModelVersion describing the finalized model version
+        """
+        req_body = message_to_json(FinalizeModelVersionRequest(name=name, version=version))
+        return self._call_endpoint(FinalizeModelVersionRequest, req_body).model_version
+
+    def _get_temporary_model_version_write_credentials(self, name, version):
+        """
+        Get temporary credentials for uploading model version files
+        :param name:
+        :param version:
+        :return:
```

Comment:
```suggestion
        :return: ...
```

Return value is missing.